bin/miden-tx-prover/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "miden-tx-prover"
+version = "0.6.0"
+description = "Miden rollup transaction prover"
+readme = "README.md"
+categories = ["no-std"]
+keywords = ["miden", "transaction"]
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+
+[[bin]]
+name = "miden-tx-prover-service"
+path = "src/bin.rs"
+
+[features]
+default = ["std"]
+std = []
+testing = ["miden-objects/testing", "miden-lib/testing", "miden-tx/testing"]
+async = []
+
+[dependencies]
+miden-lib = { workspace = true }
+miden-objects = { workspace = true }
+miden-prover = { workspace = true }
+miden-verifier = { workspace = true }
+miden-tx = { workspace = true }
+rand = { workspace = true }
+rand_chacha = { version = "0.3", default-features = false, optional = true }
+vm-processor = { workspace = true }
+winter-maybe-async = { version = "0.10" }
+axum = "0.7.5"
+reqwest = { version = "0.11", features = ["json"] }
+serde = { version = "1.0", features = ["derive"] }
+tokio = { version = "1.38", features = ["full"] }
+tonic = "0.11"
+prost = "0.12"
+tonic-web = { version = "0.11" }
+tokio-stream = { version = "0.1", features = [ "net" ]}
+tracing = { version = "0.1" }
+tracing-subscriber = { version = "0.3", features = ["fmt",  "json",  "env-filter"] }
```

Comment:
nit: let's alphabetize these.

Also: do we actually need all of these dependencies? e.g., what is `miden-verifier` for?

Reply:
No, I think that half of this can be safely removed. Will do.